easybuild/easyconfigs/p/pymatgen/pymatgen-2023.3.10-foss-2022a.eb

```diff
@@ -0,0 +1,81 @@
+easyblock = 'PythonBundle'
+
+name = 'pymatgen'
+version = '2023.3.10'
+
+homepage = 'https://pymatgen.org/'
+description = """Python Materials Genomics is a robust materials analysis code that defines core object
+ representations for structures and molecules with support for many electronic structure codes."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+builddependencies = [
+    ('hypothesis', '6.46.7'),  # required for numpy tests
+]
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('PyYAML', '6.0'),
+    ('matplotlib', '3.5.2'),
+    ('networkx', '2.8.4'),
+    ('sympy', '1.11.1'),
```

Comment:
A different version of Sympy is already used in this toolchain (see CI failure). Can we adapt to that? Or does this require 1.11.1 specifically?